spec/support/pages/forecast_page.rb

```diff
@@ -22,13 +14,11 @@ def cancel
       root_element.find('button', text: 'cancel').click
     end
 
-    def click
-      root_element.click
-    end
+    delegate :click, to: :root_element
   end
 
   set_url '/v3#/forecast'
-  set_url_matcher /\/v3#\/forecast$/
+  set_url_matcher %r{/v3#/forecast$}
 
-  sections :projected_entries, ProjectedEntrySection, ".table__rows"
+  sections :projected_entries, ProjectedEntrySection, '.table__rows'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.